docs/core/porting/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Port from .NET Framework to .NET 5
-description: Understand the porting process and discover tools you may find helpful when porting a .NET Framework project to .NET 5 (and .NET Core 3.1).
+title: Port from .NET Framework to .NET 6
```

Comment:
Does it make sense to mention only ".NET", or ".NET 5+"?

Reply:
@Youssef1313 We'll eventually revisit these titles when the manager in charge of this content gets back to restructuring/editing/adding stuff. For now I think we'll just call out 6 directly.